packages/terra-theme-provider/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Changed default behavior to always apply theme globally
```

Comment:
Can you update this comment to be a bit more descriptive. I'm not sure what this means without looking at the code changes

Reply:
Updated the changlog entry to be more descriptive: https://github.com/cerner/terra-framework/pull/1031/commits/0c05483821789068554300b3945aaf7198b959be